.wordlist.txt

```diff
@@ -19,6 +19,8 @@ async
 autoplay
 aZRf
 blockquote
+brighttalk
+BrightTalk
```

Comment:
Total nitpick: Note the vendor calls themselves BrightTALK, wonder if we should align to that ?

![image](https://user-images.githubusercontent.com/33203301/110337593-18d08a80-7ff4-11eb-817d-2849d3165e46.png)